Content.Server/GameObjects/Components/Movement/ClimbModeComponent.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Robust.Shared.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Maths;
+using Robust.Shared.GameObjects.Components;
+using Content.Shared.Physics;
+using Content.Shared.Maps;
+using Content.Shared.GameObjects.Components.Movement;
+using Content.Shared.GameObjects.EntitySystems;
+
+namespace Content.Server.GameObjects.Components.Movement
+{
+    [RegisterComponent]
+    public class ClimbModeComponent : SharedClimbModeComponent, IActionBlocker
```

Comment:
Just a nitpick: A better name in my opinion would be `ClimbingComponent`

Reply:
Yeah i struggled to figure out the proper name. Components for verbs maybe need a standardized way to name them, i think [Verb]ingComponent is probably good. Could even make it more verbose to delineate that it's a verb, it could be like [Verb]VerbComponent

I'll leave it to everyone else